client/webserver/locales/pt-br.go

```diff
@@ -0,0 +1,171 @@
+package locales
+
+var PtBr = map[string]string{
+	"Language":              "pt-BR",
+	"Markets":               "Mercados",
+	"Wallets":               "Carteiras",
```

Comment:
Would you add a map entry for `Export Trades` please.

https://github.com/decred/dcrdex/blob/6e6cac4bf16f42ee6ad9e47721bb15263874f22d/client/webserver/site/src/html/orders.tmpl#L39